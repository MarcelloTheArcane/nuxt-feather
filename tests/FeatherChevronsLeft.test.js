import Vue from 'vue/dist/vue'

import FeatherChevronsLeft from '../icons/FeatherChevronsLeft.vue'

describe('FeatherChevronsLeft component', () => {
  const vm = new Vue(FeatherChevronsLeft).$mount()

  it('is an SVG element', () => {
    expect(vm.$el.tagName).toBe('svg')
  })

  it('matches snapshot', () => {
    expect(vm.$el).toMatchSnapshot()
  })

  it('defaults to stroke width of 2', () => {
    expect(vm.$el.getAttribute('stroke-width')).toBe('2')
  })

  it('has no children with stroke width or colour properties', () => {
    for (const child of vm.$el.children) {
      expect(child.getAttribute('stroke-width')).toBeNull()
      expect(child.getAttribute('stroke')).toBeNull()
    }
  })
})
