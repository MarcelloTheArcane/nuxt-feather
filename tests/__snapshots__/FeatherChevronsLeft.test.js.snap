// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeatherChevronsLeft component matches snapshot 1`] = `
<svg
  fill="none"
  height="24"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="11 17 6 12 11 7"
  />
   
  <polyline
    points="18 17 13 12 18 7"
  />
</svg>
`;
